internal/bin/cldev

```diff
@@ -16,7 +16,9 @@ source internal/bin/clenv
 export ROOT=$PROJECT_ROOT/internal/clroot
 
 mainexec() {
-  go run -ldflags "$LDFLAGS" main.go $@
+  mkdir -p tmp
```

Comment:
Should use `mktemp` with a `trap`

Reply:
`mktemp -p` shouldn't raise a signal. Hopefully then we can avoid `trap`.

Reply:
the `trap` is for cleaning up!